src/vuestic-theme/vuestic-components/va-card/VaCard.vue

```diff
@@ -0,0 +1,240 @@
+<template>
+  <div
+    class="va-card"
+    :class="computedCardClass"
+  >
+    <div class="va-card__image" v-if="image">
+      <img :src="image" :alt="imageAlt">
+      <div class="va-card__image-overlay" v-if="overlay"></div>
+    </div>
+
+    <div
+      class="va-card__header"
+      :class="{'va-card__header--over': image && titleOnImage}"
+      v-if="showHeader"
+    >
+      <div class="va-card__header-inner">
+        <slot name="header">
+          <div class="va-card__header-title">
+            {{ title }}
+          </div>
+          <div class="va-card__header-actions">
+            <slot name="actions" />
+          </div>
+        </slot>
+      </div>
+    </div>
+
+    <div
+      class="va-card__body"
+      :class="computedCardBodyClass"
+    >
+      <slot/>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'va-card',
+  props: {
+    stripe: {
+      type: String,
+      default: '',
+    },
+    title: {
+      type: String,
+      default: '',
+    },
+    theme: {
+      type: String,
+      validator: (val) => ['base', 'bright', 'dark'].includes(val),
+      default: 'base',
+    },
+    noPaddingV: {
+      type: Boolean,
+      default: false,
+    },
+    noPadding: {
+      type: Boolean,
+      default: false,
+    },
+    image: {
+      type: String,
+      default: '',
+    },
+    overlay: {
+      type: Boolean,
+      default: false,
+    },
+    imageAlt: {
+      type: String,
+      default: '',
+    },
+    titleOnImage: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  computed: {
+    showHeader () {
+      return this.title || this.$slots.header || this.$slots.actions
+    },
+    computedCardClass () {
+      return {
+        'va-card--theme-dark': this.theme === 'dark',
+        'va-card--theme-bright': this.theme === 'bright',
+        [`va-card--stripe-${this.stripe}`]: this.stripe
+      }
+    },
+    computedCardBodyClass () {
+      return {
+        'pt-0': this.noPaddingV,
```

Comment:
No global classes in UI Components.

> If you work on UI components - work in book environment (yarn serve:book). We want to keep global stuff out of components.

From [guide](https://github.com/epicmaxco/vuestic-admin/blob/master/.github/CONTRIBUTING.md)